src/librustdoc/clean/inline.rs

```diff
@@ -623,6 +621,12 @@ crate fn record_extern_trait(cx: &DocContext<'_>, did: DefId) {
     debug!("record_extern_trait: {:?}", did);
     let trait_ = build_external_trait(cx, did);
 
-    cx.external_traits.borrow_mut().insert(did, trait_);
+    cx.external_traits.borrow_mut().insert(
+        did,
+        clean::TraitWithExtraInfo {
+            trait_,
+            is_spotlight: clean::utils::has_doc_flag(cx.tcx.get_attrs(did), sym::spotlight),
+        },
+    );
```

Comment:
nit: rustfmt really mangles this formatting, do you mind fixing it manually?
```suggestion
    let trait_ = clean::TraitWithExtraInfo {
        trait_,
        is_spotlight: clean::utils::has_doc_flag(cx.tcx.get_attrs(did), sym::spotlight),
    };
    cx.external_traits.borrow_mut().insert(did, trait_);
```

Reply:
Sure.